refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/resources/application-local.yml

```
@@ -4,15 +4,18 @@ spring:
+server:
+  port: 8086
+
 refarch:
   s3:
     bucket-name: test-bucket
     access-key: minio
     secret-key: Test1234
-    url: http://localhost:9000
-server:
-  port: 8086
+    url: http://localhost:9000
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)